### DISCUSSION TITLE
Login loop

### DISCUSSION THREAD

**Discussion by 0xABCDE** [Budibase/budibase#11652] [Help]
Hi there,

I have latest Budibase setup via docker (portainer-self hosted). Upon login, the "`Logged-in Successfully`" message is shown but the user is not actually logged in. Browser console shows the error `HTTP 403 on GET:https://mydomain/api/global/self - Unauthorized`.

What could possibly cause this?  I set up Budibase via teleport authentication, which probably caused this behavior. Are there any headers checked during authentication that will prevent log-in?

Thanks

**Comment by melohagan** [comment]
Hi @0xABCDE 

I am unfamiliar with teleport authentication. Does it support OIDC?

**Reply by 0xABCDE** [reply]
Hi @melohagan , thank you for taking time to reply. 

Apparently it does [https://goteleport.com/docs/access-controls/sso/oidc/]. Should this be implemented to overcome the login loop problem?

**Reply by melohagan** [reply]
Possibly related to this: https://github.com/Budibase/budibase/issues/11673

Here is the OIDC docs, to double check you have everything configured correctly: https://docs.budibase.com/docs/openid-connect

**Reply by 0xABCDE** [reply]
I see. I will check up on this. however, implementing OIDC is not mandatory right? Teleport merely provides secure connection to the Budibase docker container which is not open to the [public internet. Like what a VPN does. Apart from OIDC configuration, could there be any situation in which console shows 403 - Unauthorized while the standard Budibase credentials are correct (and displays Login Successful)
?
